components/builder-api-client/src/builder.rs

```diff
@@ -473,24 +482,24 @@ impl BuilderAPIProvider for BuilderAPIClient {
                             key: &str,
                             secret: &WrappedSealedBox)
                             -> Result<()> {
+        debug!("Creating origin secret: {}, {}", origin, key);
```

Comment:
Seeing "key" in a logging statement (in a function named `create_origin_secret`, no less) got me concerned about leaking confidential information in logs.

It looks like "key" in this context refers to a hash map key though, so that's fine for logging. However, it might be good to change the argument name from `key` to something a bit more descriptive (and less concerning!) like `name`.

Reply:
OK